frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/index.tsx

```diff
@@ -227,6 +229,8 @@ const DatasetRow: FC<Props> = ({
               dataAssets={dataset?.dataset_assets}
               Button={DownloadButton}
               isDisabled={dataset.tombstone}
+              // rdsSkipped is drilled 3 components down to `frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/index.tsx`
+              rdsSkipped={rdsSkipped}
```

Comment:
I don't think we need this, per my comment above. Personally I think we should just use `availableFormats` in the relevant spot and then we don't to add this whole separate parameter.

Reply:
Irrelevant per above discussion